DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -22,18 +22,17 @@ function Get-TargetResource
         [ValidateNotNullOrEmpty()]
         [string]
         $Name,
-        [ValidateNotNullOrEmpty()]
         [string]
         $Params,
-        [ValidateNotNullOrEmpty()]
-        [string]
+        [string[]]
         $Version,
-        [ValidateNotNullOrEmpty()]

```

Comment:
Why are the `[ValidateNotNullOrEmpty()]` being removed?

Reply:
Because this parameters are optional. And not required. For example, if you don't set version number, check will be not for specific version, but for existing program of any version.

Reply:
`[ValidateNotNullOrEmpty()]` doesn't make the parameter mandatory. It checks any parameter that is passed to it to ensure it's not null or empty.